src/modules/core/components/NotEnoughReputation/NotEnoughReputation.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { defineMessage, FormattedMessage } from 'react-intl';
+
+import styles from './NotEnoughReputation.css';
+
+const MSG = defineMessage({
+  title: {
+    id: 'NotEnoughReputation.title',
+    defaultMessage: 'You don’t have enough reputation to create this action.',
+  },
+  description: {
+    id: 'NotEnoughReputation.description',
+    defaultMessage: `If you would like to enforce the action you can use
+    permissions. To create the action toggle on Force option in top
+    right corner.`,
+  },
+});
+
+const displayName = 'NotEnoughReputation';
+
+const NotEnoughReputation = () => (
```

Comment:
I don't think this should be a core component as it's not general enough.

So either move it out of core, or tell my why you think it should remain in here :)

Reply:
moved to dashboard